test/test.js

```diff
@@ -186,6 +186,32 @@ describe('sanitizeHtml', function() {
             ''
         );
   });
+  it('Should not collapse elements containing only self closing tags', function() {
```

Comment:
This doesn't quite test for what it says it does. It doesn't matter if the tag is self closing. That can be fixed with a pretty small change in `index.js`, moving `frame.updateParentNodeChildren();` into the `options.Selfclosing` check below where it is now.